packages/payment-detection/src/thegraph/index.ts

```diff
@@ -31,7 +31,7 @@ export const getTheGraphClient = (
 
 // Note: temporary until TheGraph has been thoroughly tested
 export const networkSupportsTheGraph = (network: string): boolean => {
-  return !['mainnet', 'rinkeby', 'private', 'fantom'].includes(network);
+  return !['mainnet', 'rinkeby', 'private', 'fantom', 'arbitrum-rinkeby'].includes(network);
```

Comment:
is this temporary? all new networks should be based on the graph if possible

Reply:
The subgraph for arbitrum-rinkeby was not deployed before, but now that it is I'll change this